about.html

```diff
@@ -57,23 +65,23 @@ <h5 class="venue">Please Contact : globalhealth@gmail.com <br>Mirpur national st
     <section class="about-section">
         <section class="about-section-a">
             <article>
-             <h2 class="about-creative-title">Creative Commons <br> Global Summit 2015 Logo</h2>
-             <img src="#" alt="">
+             <h2 class="about-creative-title">Creative Commons Global Summit 2015 Logo</h2>
+             <img src="#" alt="image">
```

Comment:
- [ ] [OPTIONAL] I would recommend using a descriptive `alt` name, for example, you can describe the image in two or three words, this way when the image is inaccessible or not loading for some reason the user knows what it's about, also it's good for accessibility like screen readers